Adding 100 active placement FAQ (#9815)
braze-inc/braze-docs: _includes/banners/faq.md

For more information, see [Placement requests]({% if include.section == "user" %}{{site.baseurl}}/user_guide/message_building_by_channel/banners#requests{% elsif include.section == "developer" %}{{site.baseurl}}/developer_guide/banners#requests{% endif %}).

## How many Banner campaigns can be active simultaneously?

> Each workspace supports a maximum of **100 active Banner campaigns** at any one time. If you reach this limit, you'll need to archive or deactivate older campaigns before creating new ones.

Reviewer: ```suggestion
Each workspace can support up to 100 active Banner campaigns. If this limit is reached, you'll need to [archive or deactivate](https://www.braze.com/docs/user_guide/engagement_tools/messaging_fundamentals/about_statuses/#changing-the-status) an existing campaign before creating a new one.
```

> Each workspace can support up to 100 active Banner campaigns. If this limit is reached, you'll need to [archive or deactivate](https://www.braze.com/docs/user_guide/engagement_tools/messaging_fundamentals/about_statuses/#changing-the-status) an existing campaign before creating a new one.

Reviewer: ```suggestion
Each workspace can support up to 100 active Banner campaigns. If this limit is reached, you'll need to [archive or deactivate]({{site.baseurl}}/user_guide/engagement_tools/messaging_fundamentals/about_statuses/#changing-the-status) an existing campaign before creating a new one.
```
